### DISCUSSION TITLE
Is it possible to configure Kitty to not show the window width and height when resizing?

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#6487] [Q&A] [answered]
When I resize the window, the updated width and height are displayed, which causes a slight visual distortion when creating new os windows in the same space on MacOS.

Here's a video of it:

https://github.com/kovidgoyal/kitty/assets/75550932/77398a58-8cfb-404f-b952-90608a810ed9

I've found this thread: https://github.com/kovidgoyal/kitty/discussions/6357, but I can't find the `resize_window_strategy` option.

**Comment by kovidgoyal** [comment]
No it isnt that option was removed. I dont see a distortion, unless by distortion you mean the size is momentarily displayed.

**Reply by ghost** [reply]
You're right, it isn't really a distortion. The video doesn't show it like it's visible on my monitor. And yes, I meant the size being displayed momentarily. Is there a way to disable showing the size?

**Comment by kovidgoyal** [comment, marked as answer]
See if that https://github.com/kovidgoyal/kitty/commit/601e502d2e27baca7446587cb5188c965eaf3789 helps

**Reply by ghost** [reply]
Sorry for the late reply. A build from that commit solved it. Is this change going to be included in the next release?
